api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -791,4 +791,26 @@ public void getCountOfDrugs_shouldReturnTheTotalNumberOfMatchingNumbers() {
 		assertEquals(new Integer(1),
 		    conceptService.getCountOfDrugs(phrase, conceptService.getConcept(conceptId), true, true, true));
 	}
+
+	/**
+	 * @see ConceptServiceImpl#getConceptNameTagByName(String)
+	 */
+	@Test
+	public void getConceptNameTagByName_shouldReturnNullIfNoConceptNameTagIsFound() {
+		assertNull(conceptService.getConceptNameTagByName(null));
```

Comment:
How to do ensure that by requesting `.getConceptNameTagByName(null)` for should return null if No concept nameTag is found?

Reply:
[This](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/db/hibernate/HibernateConceptDAO.java#L873) method here explains that and also [this](https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java#L732) test case shares similar behaviour

Reply:
A 'null' tag will obviously return null for .getConceptNameTagByName(null) How about you try passing a tag value (not null) which is not present in the sampleDataset to see how the test actually works

Reply:
Good idea, I'll work on it :)